app/src/main/java/de/westnordost/streetcomplete/quests/lamp_type/AddLampType.kt

```diff
@@ -16,8 +16,8 @@ class AddLampType : OsmFilterQuestType<String>() {
     override val elementFilter = """
         nodes with
           highway = street_lamp
-          and !lamp_type
-          and !light:method
+          and ( !lamp_type or lamp_type ~ electric|floodlight|sodium|solar_lamp )
+          and ( !light:method or light:method ~ electric|discharge|sodium )
```

Comment:
Formatting suggestion to be consistent with SC style:

```suggestion
          and (!lamp_type or lamp_type ~ electric|floodlight|sodium|solar_lamp)
          and (!light:method or light:method ~ electric|discharge|sodium)
```